modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -416,20 +416,17 @@ def __update_node_log_file(self, node):
             self.logger.debug(f"rotating {node.log_file} to {rotated_log} on {node.name}")
             node.account.ssh(f"mv {node.log_file} {rotated_log}")
 
-    def _update_ssl_config_with_globals(self, dict_name: str, default_jks: str):
+    def _update_ssl_config_with_globals(self, dict_name: str):
         """
         Update ssl configuration.
         """
-        _dict = self.globals.get(dict_name)
-
-        if _dict is not None:
-            ssl_context_factory = SslContextFactory(self.install_root, **_dict)
-        else:
-            ssl_context_factory = SslContextFactory(self.install_root, default_jks)
-
-        self.config = self.config._replace(ssl_context_factory=ssl_context_factory)
-        self.config = self.config._replace(connector_configuration=ConnectorConfiguration(
-            ssl_enabled=True, ssl_context_factory=ssl_context_factory))
+        ssl_params = None
+        if self.config.ssl_params is None:
+            ssl_params = get_ssl_params_from_globals(self.globals, dict_name)
+        if ssl_params:
+            self.config = self.config._replace(ssl_params=ssl_params)
+            self.config = self.config._replace(connector_configuration=ConnectorConfiguration(
+                ssl_enabled=True, ssl_params=ssl_params))
```

Comment:
Why we set `ssl-params` to two places?

Reply:
We have SSL params in IgniteConfiguration and in ConnectorConfiguration, so we should change them both